sql-plugin/src/main/java/com/nvidia/spark/rapids/iceberg/spark/source/GpuSparkScan.java

```diff
@@ -282,7 +282,7 @@ private static class MultiFileBatchReader
                          boolean queryUsesInputFile) {
       super(task.task, task.table(), task.expectedSchema(), task.isCaseSensitive(),
           task.getConfiguration(), task.getMaxBatchSizeRows(), task.getMaxBatchSizeBytes(),
-          task.getTargetBatchSizeBytes(), task.useChunkedReader(),
+          task.getTargetBatchSizeBytes(), task.getMaxBatchSizeBytes(), task.useChunkedReader(),
```

Comment:
This is max batch size instead of max column size?

Reply:
Replaced with `getMaxGpuColumnSizeBytes()`